azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -647,7 +647,6 @@ def __init__(self):
         self.retry_intervals = {
             pipeline_ops_mqtt.MQTTSubscribeOperation: 20,
             pipeline_ops_mqtt.MQTTUnsubscribeOperation: 20,
-            pipeline_ops_base.ConnectOperation: 20,
```

Comment:
>pipeline_ops_base.ConnectOperation: 20, [](start = 12, length = 39)

This is a very significant change.  The RetryStage is no longer retrying ConnectOperation.  That's entirely the responsibility of the ReconnectStage.  Without this change, it's impossible to reason about the behavior of the pipeline because there would be two different stages that are both responsible for reconnecting depending on how the error happens.

Reply:
Add this information to the class docstring perhaps?

---
In reply to: [377748902](https://github.com/Azure/azure-iot-sdk-python/pull/460#discussion_r377748902) [](ancestors = 377748902)